### ISSUE TITLE
Wireless card issue on Pixelbook 2017

### ISSUE THREAD

**Issue by Gameparkwing** [MrChromebox/firmware#657]
The Google Pixelbook 2017 (EVE) is device I am using.
I had been upgraded the firmware to MrChromebox (verison 4.22.4 Release) and installed Windows 11.
The system is working normally, but the status of wireless network card has a issue.
Wireless card can be working, issue was it appears in removable devices menu.

![fe57b8f6ddaff5c6323bfc30e722dd7a570a2778](https://github.com/user-attachments/assets/2dbab833-6178-4ca0-ada2-e1b9dce930eb)

**Comment by MrChromebox** [comment]
first I've heard of that on a Pixelbook, no idea why it would appear that way. There is nothing in the firmware programming to mark it as a removable device

**Comment by Gameparkwing** [comment]
Could this be related to the hotplug state of the PCIe device?
There are supports hotplug switch for PCIe in UEFI BIOS of General PC.

**Comment by l1k** [comment]
The Root Port to which the wifi card is attached is hotplug-capable:
https://github.com/MrChromebox/coreboot/blob/MrChromebox-2512/src/mainboard/google/eve/devicetree.cb#L343

That's likely the reason why the wifi card is shown as removable.

**Comment by MrChromebox** [comment]
> That's likely the reason why the wifi card is shown as removable.

is it a m.2 socketed card or soldered to the mainboard? I'm assuming the latter given EVE's packaging, in which case hotplug should not be enabled, and it should be marked as built-in

**Comment by Gameparkwing** [comment]
Yes, I watch the disassembly and found that wireless card was soldered to the mainboard.
So "PcieRpHotPlug[0]" for PCIe Root Port 1 should be set to '0'?

**Comment by MrChromebox** [comment]
> Yes, I watch the disassembly and found that wireless card was soldered to the mainboard. So "PcieRpHotPlug[0]" for PCIe Root Port 1 should be set to '0'?

yes but I'm not sure that will change Windows behavior. wouldn't hurt to try though in conjunction with the actual fix for the ASPM

**Comment by Gameparkwing** [comment]
I don't have a build environment. Can you build a firmware for me to testing?

**Comment by MrChromebox** [comment]
@Gameparkwing use the latest release firmware and let me know if there's still an issue

**Comment by Gameparkwing** [comment]
> [@Gameparkwing](https://github.com/Gameparkwing) use the latest release firmware and let me know if there's still an issue

I has been upgraded firmware to (2026.07.14) and issue was stilled.
